packages/swapper/src/swappers/zrx/utils/test-data/setupSwapQuote.ts

```diff
@@ -16,7 +16,14 @@ export const setupQuote = () => {
     receiveAddress: 'receiveAddress',
     sellAssetAccountId: '0',
     buyAssetAccountId: '0',
-    sendMax: false
+    sendMax: false,
+    minimum: '0',
+    maximum: '999999999999',
```

Comment:
```suggestion
    maximum: Number.MAX_SAFE_INTEGER.toString(),
```

don't use magic numbers - i'm sure there's a bunch of near infinite supply shitcoins that you could conceivable sell more than this for a few dollars

Reply:
oh i see this is test data, comment still applies re magic numbers

Reply:
True. 

its also just test data but im fine with making that change